src/ol/object.js

```diff
@@ -223,12 +230,15 @@ ol.Object.prototype.setProperties = function(values) {
 /**
  * Unsets a property.
  * @param {string} key Key name.
+ * @param {boolean=} opt_silent update propertie silently
  * @api stable
  */
-ol.Object.prototype.unset = function(key) {
+ol.Object.prototype.unset = function(key, opt_silent) {
   if (key in this.values_) {
     var oldValue = this.values_[key];
     delete this.values_[key];
-    this.notify(key, oldValue);
+    if (!goog.isDef(opt_silent) || opt_silent !== true) {
```

Comment:
Same as above regarding the use of `goog.isDef()`.  It is enough to check `!opt_silent`.